src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,25 @@
+$cell-size: 100px;
+
+$gap: $cell-size * 0.01;
+$padding: $cell-size * 0.1;
+$calendar-width: 7 * $cell-size + 6 * $gap;
+$cell-shift: $cell-size + $gap;
+
+$font-size: $cell-size * 0.3;
+
+$cell-background-color: #eee;
+$cell-background-color-hover: pink;
+$cell-transform-hover: $cell-size * 0.2;
+
+$border-width: $cell-size * 0.01;
+$border-style: solid;
+$border-color: black;
+
+$week:
+  "mon" 0,
+  "tue" $cell-shift,
+  "wed" 2 * $cell-shift,
+  "thu" 3 * $cell-shift,
+  "fri" 4 * $cell-shift,
+  "sat" 5 * $cell-shift,
+  "sun" 6 * $cell-shift;
```

Comment:
you shouldn't repeat such formula many times
store only day name and day number here, go through this variable in cycle and use correct formula inside the cycle